python/src/NearestPointProblem_doc.i.in

```diff
@@ -1,13 +1,21 @@
 %feature("docstring") OT::NearestPointProblem
 "Nearest point problem.
 
-This represents a nearest point problem:
+This defines a nearest point problem:
 
 .. math::
 
-    \min_{x} \frac{1}{2}||x||^2 \\
+    \min_{x} \frac{1}{2}\|x\|_2^2 \\
     g(x) = v
 
+where :math:`\| \cdo t\|_2` is the euclidian norm. 
+
+In other words, this is a minimum norm problem 
+with a general nonlinear constraint. 
+
+Available constructors:
+    NearestPointProblem(*level, value*)
```

Comment:
Dont list the ctors when there's only one available.

Reply:
I see the reasoning, but I think that any class should have its constructors documented: doing so enables to just copy/paste the constructor into the code. Are there other classes with the same property?

Reply:
Yes, a lot!

Reply:
Ok: will fix this.

Reply:
Done at d9d32b2de6295f4d6f51fc583c7fdeff032cbfaf